spec/index.bs

```diff
@@ -513,6 +508,29 @@ interface mixin GPUObjectBase {
         An internal slot holding the [=device=] which owns the [=internal object=].
 </dl>
 
+{{GPUObjectDescriptorBase}} has the following members:
+
+<dl dfn-type=dict-member dfn-for=GPUObjectDescriptorBase>
+    : <dfn>label</dfn>
+    ::
+        The initial value of {{GPUObjectBase/label|GPUObjectBase.label}}.
+</dl>
+
+
+: <dfn dfn>Debug label</dfn>
+:: A string provided for use by development tools (such as error/warning messages,
```

Comment:
read slightly ambiguous: is the string provided by the dev tools, or used by the dev tools. May need to be reworded slightly